backend/README.md

```diff
@@ -0,0 +1,49 @@
+## Run locally
+
+You'll need access to the Firebase project at https://console.firebase.google.com/project/c19-backend.
```

Comment:
Are only users with a "recidiviz.org" email address allowed to access this URL?  I tried with my normal Gmail account and received a "Your operation was forbidden" error.

Reply:
FYI, this is when I try to access this URL via a browser.

Reply:
As discussed, access is restricted to only invited admins (at the moment: Josh, me, and now you).